app/views/bus_stops/by_route.haml

```diff
@@ -3,14 +3,10 @@
 - if @stops.empty?
   No stops on this route
 - else
-  - @stops.each_pair do |complete, stops|
-    - if complete
-      Completed
-    - else
-      Not Completed
-    Stops
+  - @stops_hash.each_pair do |status, stops|
+    = status.humanize + ' Stops'
```

Comment:
But again, why do this translation? Why not just put the string that we want into the view as the keys in the hash?